recipes/heavydb/meta.yaml

```
@@ -0,0 +1,575 @@
+        - libthrift {{ thrift_version }}
+        - pyarrow {{ arrow_version }}
+        - python
+        - tbb
+      run_exports:
+        - {{ pin_subpackage('pyheavydbe',  max_pin='x.x.x') }}
+    requirements:
+      build:
+        # c compiler is specified here to get run constraint pins correct, presumably..
+        - {{ compiler('c') }}
+        - {{ compiler('cgo') }}
+        - {{ compiler('cxx') }}
+        - {{ compiler("cuda") }}  # [cuda_compiler_version not in (undefined, "None")]
+        - clangdev {{ llvm_version }}
+        # clang++ is used for generating the bytecodes of extension functions
+        - clangxx {{ llvm_version }}
+        - cmake
+        - llvmdev {{ llvm_version }}
+        - make
+        - ninja
+        - maven
+      host:
+        - arrow-cpp ={{ arrow_version }}=*{{ build_ext }}
+        - bisonpp
+        - blosc
+        - boost-cpp
+        - clangdev {{ llvm_version }}
+        - cython
+        - double-conversion
+        - flex
+        - fmt
+        - glog
+        - llvmdev {{ llvm_version }}
+        - llvm {{ llvm_version }}
+        - libarchive
+        - libgdal
+        - libpng
+        - librdkafka
+        - numpy
+        - pip
+        - pyarrow ={{ arrow_version }}=*{{ build_ext }}
+        - python
+        - pytest
+        - tbb
+        - tbb-devel
+        - thrift-cpp {{ thrift_version }}
+        - {{ pin_subpackage('heavydbe', exact=True) }}
+      run:
+        - arrow-cpp-proc {{ arrow_proc_version }} {{ build_ext }}
+        - pyarrow ={{ arrow_version }}=*{{ build_ext }}
+        - python
+        - tbb4py
+        - {{ pin_subpackage('heavydbe', exact=True) }}
+    test:
+      requires:
+        - pytest
+        - numpy
+        - pandas
+      imports:
+        - heavydbe
+      source_files:
+        - Embedded/test/test_exceptions.py
+      commands:
+        - pytest -sv Embedded/test/test_exceptions.py
+
+    about:
+      home: https://heavy.ai
+      license: Apache-2.0
+      license_family: APACHE
+      license_file: LICENSE.md
+      summary: The HeavyDB database
+
+      description: |
+        HeavyDB is an in-memory, column store, SQL relational database
+        that was designed from the ground up to run on GPUs.
+      doc_url: https://docs.heavy.ai/
+      dev_url: https://github.com/heavyai/heavydb
+
+about:
+  home: https://heavy.ai
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE.md
+  summary: The HeavyDB database
+
+  description: |
+    HeavyDB is an in-memory, column store, SQL relational database
+    that was designed from the ground up to run on GPUs.
+
+    This recipe provides the following packages:
+      heavydb-common CUDA-enabled heavydb, cpu and cuda builds
+      pyheavydb Python connector
+      heavydbe embedding library, cpu and cuda builds
+      pyheavydbe-embedded Python extension module
+  doc_url: https://docs.heavy.ai/
+  dev_url: https://github.com/heavyai/heavydb
+
+extra:
```

```suggestion
extra:
  feedstock-name: heavydb
```

So you get a nice feedstock name :)